packages/core/testing/src/test_bed.ts

```diff
@@ -58,7 +58,7 @@ export interface TestBed {
 
   get<T>(token: Type<T>|InjectionToken<T>, notFoundValue?: T, flags?: InjectFlags): any;
   /**
-   * @deprecated from v8.0.0 use Type<T> or InjectionToken<T>
+   * deprecated from v8.0.0 use Type<T> or InjectionToken<T>
```

Comment:
Can we add a small note here about why we aren't using the official @deprecated? I could see someone "fixing" it later by accident.

```suggestion
// TODO: switch back to official deprecation marker once TSLint issue is resolved
/**
 * deprecated from v8.0.0 use Type<T> or InjectionToken<T>
```